api/working-with-extensions/continuous-integration.md

```diff
@@ -11,6 +11,18 @@ MetaDescription: Use Continuous Integration for testing Visual Studio Code exten
 
 Extension integration tests can be run on CI services. The [`vscode-test`](https://github.com/Microsoft/vscode-test) library helps you setup extension tests on CI providers and contains a [sample extension](https://github.com/microsoft/vscode-test/tree/master/sample) setup on Azure Pipelines. You can check out the [build pipeline](https://dev.azure.com/vscode/vscode-test/_build?definitionId=15) or jump directly to the [`azure-pipelines.yml` file](https://github.com/microsoft/vscode-test/blob/master/sample/azure-pipelines.yml).
 
+## Automated Publishing
+
+You can also configure the CI to publish a new version of the extension automatically.
+
+The publish command is similar to publishing from a local environment using the [`vsce`](https://github.com/Microsoft/vsce) service but the command needs to also include the Personal Access Token (PAT). [`vsce`](https://github.com/Microsoft/vsce) can also pick up on the `VSCE_PAT` environmental variable for the Personal Access Token.
+
+You shouldn't expose the PAT with the rest of the source code (it's a sensitive information), so you can store it in a "secret variable". The value of that variable will not be exposed and you can use it in the CI pipline.
+
+# Example Implementations
+
+The following examples use a sample project that uses [Yarn](https://yarnpkg.com/), but can be adapted to use [npm](https://www.npmjs.com/) as well.
+
```

Comment:
I suggest to simply remove this from the start and just add an FAQ item at the end of the page.